cmd/createcluster.go

```diff
@@ -127,48 +131,102 @@ func runCreateCluster(ctx context.Context, w io.Writer, conf clusterConfig) erro
 		conf.Network = eth2util.Goerli.Name
 	}
 
-	if err := validateClusterConfig(ctx, conf); err != nil {
-		return err
+	var (
+		def            cluster.Definition
+		threshold      = conf.Threshold
```

Comment:
no need for these variables, either load def from DefFile or populate it from conf.